data/src/main/java/com/teamagam/gimelgimel/data/config/Constants.java

```diff
@@ -11,7 +11,6 @@
   public static final String MESSAGING_SERVER_DEFAULT = MESSAGING_SERVER_URL_1;
   public static final long CONNECTION_SERVER_TIME_OUT_SECONDS = 120;
 
-  //Proxy
```

Comment:
The variable names are well explained.
We use blank line to separate the categories.